src/app/components/menu/menu.ts

```diff
@@ -107,12 +109,14 @@ export class Menu implements OnDestroy {
         else
             this.show(event);
 
+        this.visibleChanged.emit(this.visible);
```

Comment:
this will fire the event twice has both method above will fire it already

Reply:
Yep that's right I've left it only in show and hide.